doc/radosgw/config-ref.rst

```diff
@@ -432,7 +432,7 @@ You may include the following settings in your Ceph configuration
 file under each ``[client.radosgw.{instance-name}]`` instance.
 
 
-``rgw zone``
+``rgw_zone``
```

Comment:
we do normalize settings names by replacing spaces with underscores. and to be more consistent with the rest of this document, i'd suggest leave as it is.

Reply:
While I agree it would be that normalized settings are good, the "rgw zone" fell back to "default" when used with 12.2.12.  Only "rgw_zone" worked.  I can re-check this with Nautilus in a couple weeks.

Reply:
@cbodley could you verify this?

Reply:
i don't understand how the config parsing logic would treat 'rgw_zone' differently from 'rgw zone'